easybuild/easyconfigs/g/GDAL/GDAL-2.2.0-intel-2017a-Python-2.7.13-HDF5-1.10.1-HDF-4.2.13.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.0'
+hdf5_ver = '1.10.1'
+hdf_ver = '4.2.13'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s-HDF-%s' % (hdf5_ver, hdf_ver)
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['0d4c326862e0f118e17418c042c2bcd037b25abd3fb198e1fc5d40b11a9fc8ea']
+patches = ['GDAL-%(version)s_fix-uchar-definition.patch']
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('netCDF', '4.4.1.1', '-HDF5-%s' % hdf5_ver),
+    ('HDF', '4.2.13', '-no-netcdf'),
```

Comment:
It's a bit silly to have a `no-netcdf` version of HDF while we include `netCDF` as a dependency anyway?

Reply:
If you look at http://trac.osgeo.org/gdal/wiki/HDF, it is explicit that the netcdf inside HDF4 needs disabling for things to work.  

Sorry not my making.

Reply:
OK w.r.t. the `-no-netcdf`, since it covers the load. But you should use `hdf_ver` here rather than hardcoding `4.2.13` to make very sure that the HDF version mentioned in the `versionsuffix` matches the only that's actually used.